src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -277,6 +277,30 @@ target_include_directories(${TARGET_NAME} SYSTEM PRIVATE $<TARGET_PROPERTY:dnnl,
 # is not (yet) needed.
 target_include_directories(${TARGET_NAME} PRIVATE $<TARGET_PROPERTY:openvino::reference,INTERFACE_INCLUDE_DIRECTORIES>)
 
+# Check if SVE is available for AARCH64; compile with that if yes.
+if (ARM OR AARCH64)
+    execute_process(
+        COMMAND lscpu
+        OUTPUT_VARIABLE CPUINFO
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+    string(FIND "${CPUINFO}" "Flags:" FLAGS_INDEX)
+    string(SUBSTRING "${CPUINFO}" ${FLAGS_INDEX} -1 CPU_FLAGS)
+    string(FIND "${CPU_FLAGS}" "sve" ISA_FOUND)
+    
+    if (NOT ISA_FOUND EQUAL -1)
+        set(HAVE_SVE ON)
+    else()
+        set(HAVE_SVE OFF)
+    endif()
+
+    if (HAVE_SVE)
+        message(STATUS "[AARCH64] ISA SVE detected")
+        target_compile_options(${TARGET_NAME} PRIVATE "-march=armv8.4-a+sve")
+        add_definitions(-DHAVE_SVE)
+    endif()
+endif()
+
 # Cross compiled function
 # TODO: The same for proposal, proposalONNX, topk
 cross_compiled_file(${TARGET_NAME}
```

Comment:
I suppose you need to add SVE target to `cross_compiled_file` function.

It will automatically compile code multiple times including SVE support. Then, in runtime SVE code will be called if processor supports it.

Please, refer to this PR https://github.com/openvinotoolkit/openvino/pull/25918 to understand how to add new modes to `cross_compiled_file`

Reply:
Thanks for providing the reference. I've incorporated the changes to the best of my understanding; I would highly appreciate your help in checking if everything has been updated correctly.

Reply:
Meanwhile, I was looking for a good way to check whether my SVE code gets called during inference. As of now, the Neon FP16 functions get called. Any idea what would be a good way to test the SVE path?

Reply:
Please, implement `with_cpu_sve`

E.g. ARM64 FP16 support is checked here
https://github.com/openvinotoolkit/openvino/blob/c158480ae3a94f9a1f4a711d06224bce934876ed/src/inference/src/system_conf.cpp#L158-L176

Reply:
Added in latest commit.

I checked the complete changelog of the Neon FP16 PR. From what I can see, I should not have missed anything else. Please let me know however.

Reply:
if it works - looks correct.